scripts/utils/starknet.py

```diff
@@ -238,13 +246,26 @@ def get_tx_url(tx_hash: int) -> str:
     return f"{NETWORK['explorer_url']}/tx/0x{tx_hash:064x}"
 
 
+def is_fixture_contract(contract_name):
+    return CONTRACTS_FIXTURES.get(contract_name) is not None
```

Comment:
This feels very heavy, I wonder if we can't find a better way to do it, maybe by modifying the object we store in `COMPILED_CONTRACTS`? Just worried that every time we want to add a folder to be compiled elsewhere than `src` or `tests`, we have to start defining `is_xxx_contract` and adding a bunch of `else` branches.

Reply:
Yeah, so for now I can see:
- fixtures { which we use for testing }
- core contracts

The idea is that all fixtures can now be supported with these new changes, if there is some other *xxx* category of contracts in the future that we need to deploy, I guess we would have to refactor it to be more modular.

What do you think?

Reply:
Ok let's do it like that for now